web/modules/custom/dcamp/dcamp.install

```diff
@@ -4,8 +4,13 @@
  *
  * Install hooks for dcamp module.
  */
+use Drupal\node\Entity\Node;
 use Psr\Log\LogLevel;
 
+function dcamp_install(){
+  _dcamp_add_dummy_sponsors_and_default_logo();
```

Comment:
The downside I see from this is that we will end up with dummy content in production. I would instead create a dcamp_demo module where we add these fixtures.